src/components/templates/PartyMap/components/Map/MapRoom.scss

```diff
@@ -50,6 +50,22 @@ $label-visible: flex;
         filter: none;
       }
     }
+
+    &--iframe {
+      z-index: z(map-room--iframe);
+
+      cursor: default;
+
+      &:hover {
+        z-index: z(map-room--iframe);
+
+        transform: none;
+
+        .maproom__image {
+          filter: none;
+        }
+      }
+    }
```

Comment:
Since these are all the same styles as `&--unclickable`, I think we should be able to combine their definitions by changing line 39 to something like this:

```scss
 &--unclickable, &--iframe {
```

Reply:
Oh.. actually.. except for the `z-index`s..

See my questions/comments/thoughts in `src/scss/constants.scss` about the layering in general. Depending on what we decide there will influence what changes (if any) to make here.

Even if we do need to keep the z-index layers seperate, we could (if we wanted) use this combined style type for the main 'shared styles', and then add the individual entries below that to just set the different `z-index`'s.

Reply:
I've renamed them `covert` and separated out `iframe` and `unclickable` for the `z-index` values